api/lib/ZwaveClient.ts

```diff
@@ -1802,7 +1798,8 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 						this.logNode(
 							zwaveNode,
 							'warn',
-							`Unable to add Node ${a.nodeId} to Group ${groupId} of ${sourceMsg}, association not allowed`,
+							// FIXME: We should explain the reason why it failed in human-readable form, see doc comments of AssociationCheckResult
```

Comment:
still TODO

Reply:
do you mean on frontend side?

Reply:
Both logs and frontend IMO

Reply:
Check what i did and let me know if it's ok for you

Reply:
Although, if we do https://github.com/zwave-js/zwave-js-ui/issues/3801, we don't have to show the error reason in the logs I think, because there shouldn't be one.

Reply:
there shouldn't doesn't mean there will not, also consider that api can be also called from driver function and MQTT so I would leave it there